tests/Fixer/AttributeNotation/OrderedAttributesFixerTest.php

```diff
@@ -71,6 +71,22 @@ public static function provideInvalidConfigurationCases(): iterable
             ],
             'The list includes attributes that are not unique.',
         ];
+
+        yield 'Wildcard in the middle of FQNs' => [
+            [
+                'sort_algorithm' => OrderedAttributesFixer::ORDER_CUSTOM,
+                'order' => ['A\*\Bar'],
+            ],
+            'Wildcards are only allowed at the end of FQNs.',
```

Comment:
given this, what about no wildcard and simply using the `order` exactly (as now) or as namespace/prefix (aka wildcard)?

Reply:
I think it makes sense to differentiate a fixed string from a wildcard to avoid ambiguity. For example, if I have a FQCN that matches the beginning of a namespace, this approach allows me to distinguish between them:

```php
[
    'A\B\C', // class
    'D\E\F',
    'A\B\C\*', // namespace
]
```

However, I did noticed that there's no need to use regex here, `str_starts_with()` is sufficient.

Reply:
But what is the rationale, why would someone want that?